client/src/Components/Layouts/NotFound/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './style.css';
+
+export default function index() {
```

Comment:
@denaHS It can be helpful to give your functional components meaningful names (for example `export default function NotFound() {` because React devtools will show that name in the console inspector (so "NotFound" will make more sense than "index")